src/Behat/Hook/Hook.php

```diff
@@ -18,4 +18,5 @@
  */
 interface Hook
 {
+    public function getFilterString(): ?string;
```

Comment:
Adding this function call is a BC break but I think we can allow it. Otherwise we cannot guarantee that a class implementing this interface will have a filterString property. I don't expect anyone to have defined other classes that implement this interface which is applied to Hook attributes

Reply:
We should remember to mention this (and Definition) in the changelog. If we hit similar things in future we should probably pull to a separate PR for visibility, but I think it's OK here.